src/main/java/momo/command/EventCommand.java

```diff
@@ -0,0 +1,55 @@
+package momo.command;
+
+import momo.InvalidCommandException;
+import momo.Storage;
+import momo.StorageException;
+import momo.task.Event;
+import momo.task.Task;
+import momo.task.TaskList;
+
+import java.time.DateTimeException;
+import java.time.LocalDate;
+
+/**
+ * Handles the functionality of adding an event, updating the {@link TaskList} and
+ * {@link Storage} to include it. It also validates the initial user input. If it was
+ * not properly formatted an {@link InvalidCommandException} is thrown which prompts
+ * user to submit a proper input
+ */
+public class EventCommand extends AddCommand {
+    public static void run(String input, Storage storage, TaskList tasks) throws InvalidCommandException,
+            StorageException {
+        try {
+            String desc = input.substring(5).trim();
```

Comment:
Consider changing the name from shorthand to its full name (i.e description). Applies to other files as well.